framework/validators/DefaultValueValidator.php

```diff
@@ -38,7 +38,15 @@ class DefaultValueValidator extends Validator
     public $skipOnEmpty = false;
 
     /**
-     * @var bool whether the [[value]] is callback.
+     * @var bool whether the [[value]] is callback. Default is `false`
+     *
+     * If `$callableValue` set `true`, [[value]] will support follow callback styles:
```

Comment:
> whether the [[value]] is callback. Default is `false`

that's enough, `callable`/`callback` explained in php manual

Reply:
tks